src/Websites/Websites/Cmdlets/AppServicePlans/NewAzureAppServicePlan.cs

```diff
@@ -63,6 +65,9 @@ public class NewAzureAppServicePlanCmdlet : AppServicePlanBaseCmdlet
         [Parameter(Mandatory = false, HelpMessage = "Run cmdlet in the background")]
         public SwitchParameter AsJob { get; set; }
 
+        [Parameter(ParameterSetName = ParameterSet1Name, Position = 7, Mandatory = false, HelpMessage = "Tags are name/value pairs that enable you to categorize resources")]
+        public Hashtable Tags { get; set; }
```

Comment:
Per design guidelines, his parameter should be called 'Tag'

Reply:
I have defined as same name as defined in the **Microsoft.Azure.Management.WebSites.Models.Resource** class. Please share the predefined guidelines. Will be helpful in future.

Reply:
@markcowl  need to clarify the usage here as a command Name vs a parameter here Tags is exposed as a parameter and not as part of the command ex. Get-AzTag? In CLI we expose this as a parameter as well & use as --tags which matches the property name. Is there some design guidelines we are missing here? Thanks!

Reply:
You *must use 'Tag as the parameter name for resource tags', Tags is not allowed. This is related to PowerShell language cmdlet parameter design guidelines around plural parameter names for collection types.  It is a standard that must be followed.